guby_backend/views.py

```diff
@@ -173,3 +173,45 @@ def event_delete(request, meetup_id, event_id):
         return redirect(f'/app/meetups/{meetup_id}/events/')
 
     return render(request, 'app/event_delete.html', {"form": form, "meetup_id": meetup_id, "event": event})
+
+@login_required()
+@has_ownership
+def speaker_add(request, meetup_id, event_id):
+    event = get_object_or_404(Event, pk=event_id)
+
+    if request.method == 'POST':
+        email = request.POST['speaker-email']
+        speaker_user = User.objects.filter(email=email)
+
+        # if username does not exists, create an inactive user and add them to speaker group
+        if speaker_user is None or len(speaker_user) == 0:
```

Comment:
I don't like this checking for length :/ 

But the alternative is

```py
User.objects.get(email=email)
```

but this throws an exception if a user doesn't exist 😒

Reply:
Yes, I did try `get` initially, but that throws an exception